docs/installation/mac.md

```diff
@@ -340,7 +338,7 @@ When you start a container it automatically shares your `/Users/username` direct
 with the VM. You can use this share point to mount directories onto your container.
 The next exercise demonstrates how to do this.
 
-1. Change to your user `$HOME` directory.
+1. Go to your user `$HOME` directory.
```

Comment:
I like to use "change" because `cd` stands for `change directory` ... tho we should probably be consistent "change into"